packages/block-library/src/block/block.json

```diff
@@ -5,7 +5,7 @@
 	"title": "Pattern",
 	"category": "reusable",
 	"description": "Create and save content to reuse across your site. Update the pattern, and the changes apply everywhere it’s used.",
- 	"keywords": [ "reusable" ],
+	"keywords": [ "reusable" ],
```

Comment:
When I merged the trunk, it included an unintended modification, but the indentation should be correct with this change 🙈